### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by jnm2**
Spotted one while looking at https://github.com/YairHalberstadt/stronginject/pull/27 and pulled up VS Code to check for more.

### CODE REVIEW

File: StrongInject.sln [hunk 1]
```diff
@@ -19,7 +19,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Utilities", "Utilities", "{EB46F000-80AB-4A94-A684-080CE86F9113}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GeneratorTestsUpdator", "Utilities\GeneratorTestsUpdator\GeneratorTestsUpdator.csproj", "{4EEDAEE4-1AAF-4918-9CA6-1EDA46256D01}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GeneratorTestsUpdater", "Utilities\GeneratorTestsUpdater\GeneratorTestsUpdater.csproj", "{4EEDAEE4-1AAF-4918-9CA6-1EDA46256D01}"
```
Comment (YairHalberstadt) [inline StrongInject.sln:22]: Have you moved the files to the correctly spelled location?

**Review by YairHalberstadt — APPROVED** [review]
Thanks for this - much appreciated!

I'll let the couple that are just conversions from British spelling to American spelling pass - clearly that's the way the way the world's going ;-)

File: StrongInject.sln (hunk 1 above)
Comment (YairHalberstadt) [reply StrongInject.sln:22]: I see you have, thanks

**Comment by jnm2** [comment]
Oh! I specifically tried to avoid doing that. Which words were they?

**Comment by YairHalberstadt** [comment]
I thought Similar vs similiar, but seems I was just wrong.

**Pull request merged**